server/src/test/java/org/spine3/server/ClientServiceShould.java

```diff
@@ -181,17 +179,17 @@ public ProjectAggregate(ProjectId id) {
 
         @Assign
         public ProjectCreated handle(CreateProject cmd, CommandContext ctx) {
-            return projectCreatedMsg(cmd.getProjectId());
+            return Given.EventMessage.projectCreatedMsg(cmd.getProjectId());
         }
 
         @Assign
         public TaskAdded handle(AddTask cmd, CommandContext ctx) {
-            return taskAddedMsg(cmd.getProjectId());
+            return Given.EventMessage.taskAddedMsg(cmd.getProjectId());
```

Comment:
And here we have duplication, too. Rename to `Given.EventMessage.taskAdded()`. 
Use the same approach in every Given class, have `Event`, `EventMessage`, `Command` and `CommandMessage` classes.